### PR TITLE
update according to terraform 0.1.0

### PR DESCRIPTION
**Pull request by shiyuhang0**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Update terraform doc according to the new terraform version https://github.com/tidbcloud/terraform-provider-tidbcloud/releases/tag/v0.1.0
1. datasource has been renamed
2. some ids are changed from `just for test` to a random number. So I just delete them in the doc.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by zhangyangyu — APPROVED** [review]

File: tidb-cloud/terraform-use-cluster-resource.md [hunk 1]
```diff
@@ -7,17 +7,17 @@ summary: Learn how to use the cluster resource to create and modify a TiDB Cloud
...
 
-In addition, you will also learn how to get the necessary information with the `tidbcloud_project` and `tidbcloud_cluster_spec` data sources.
+In addition, you will also learn how to get the necessary information with the `tidbcloud_projects` and `tidbcloud_cluster_specs` data sources.
 
 ## Prerequisites
 
 - [Get TiDB Cloud Terraform Provider](/tidb-cloud/terraform-get-tidbcloud-provider.md).
 
-## Get project IDs using the project data source
+## Get project IDs using the projects data source
```
Comment (qiancai) [inline tidb-cloud/terraform-use-cluster-resource.md:16]:
```suggestion
## Get project IDs using the `tidbcloud_projects` data source
```

Comment (zhangyangyu) [reply tidb-cloud/terraform-use-cluster-resource.md:16]: if using this style, update other data source/resource too.

**Review by ran-huang — APPROVED** [review]
LGTM

**Review by dveeden — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**